modules/ROOT/partials/configuration/schema.adoc

```diff
@@ -13,7 +13,6 @@ Also note that all event attributes are excluded by default since it's a bad pra
 
 *Possible Values:* `html5`, `html4`, `html5-strict`
```

Comment:
The line 6 contains a ref that refers to something in another file, it needs to include the full ref:
Line 6: ```xref:extended_valid_elements[`extended_valid_elements`]```